sdks/python/apache_beam/yaml/examples/spanner_read.yaml

```diff
@@ -0,0 +1,63 @@
+pipeline:
+  transforms:
+
+  # Reading data from a Spanner database. The table used here has the following columns:
+  # shipment_id (String), customer_id (String), shipment_date (String), shipment_cost (Float64), customer_name (String), customer_email (String)
+  # ReadFromSpanner transform is called using project_id, instance_id, database_id and a query
+  # A table with a list of columns can also be specified instead of a query
+    - type: ReadFromSpanner
+      name: ReadShipments
+      config:
+        project_id: 'apache-beam-testing'
+        instance_id: 'shipment-test'
+        database_id: 'shipment'
```

Comment:
@damccorm Are Spanner resources (specifically these) exposed to general public? i.e., would anyone be able to run this YAML as-is?